README.md

```diff
@@ -31,7 +31,23 @@ in `config/config.exs` would look like this:
 use Mix.Config
 
 config :my_app, MyApp.Repo,
-  loggers: [{Timber.Ecto, :log, []}]
+  log: false
+```
+
+You'll also have to attach Timber's Telemetry event handler in the Repo's `init` callback:
+
+```elixir
+# lib/my_app/repo.ex
+def init(_, opts) do
+  :ok = Telemetry.attach(
```

Comment:
Does this need to be Ecto.Telemetry?

Reply:
Nope, and it's double nice because we also don't have to add telemetry as a dependency either
https://hexdocs.pm/ecto/Ecto.Repo.html#module-telemetry-events